docs/source/community/host-guide.md

```diff
@@ -1,12 +1,8 @@
-# Host Guide 
+# **Host Guide**
 
-Welcome to the Host Guide, a place for call hosts to get help,
-info, advice and best practices for conducting engaging and effective
-meetings.
+Welcome to the Host Guide, a place for call hosts to get help, info, advice and best practices for conducting engaging and effective meetings.
 
-Check the [Suggested Meeting Script section](#meeting-script-and-checklist)
-if you want a TL;DR for quick tips to follow for running a great meeting, and
-peruse the other sections for basic info and additional context.
+Check the [Suggested Meeting Script section](https://docs.jupyter.org/en/latest/community/host-guide.html#meeting-script-and-checklist) if you want more info for running a great meeting, and peruse the other sections for basic info and additional context. This documentation can be used as a template, the host can duplicate the guide and update the links to reflect their individual meeting or simply use it as a reference point in planning. 
```

Comment:
We might want to just leave the path as it was (not a full URL) so that we don't have to worry about the URL changing